src/components/VerificationSuccessful_comp.js

```diff
@@ -0,0 +1,17 @@
+function VerificationSuccessful(){
+    return (
+        <div className="space-y-6">
+            <div className="">
+                <div className="flex justify-between ">
+                    <h1 className="text-blue-500 font-bold text-5xl">Verification Successful</h1>
+                    <h2>cancel</h2>
```

Comment:
This isn't right. There's no `cancel` there and it's a button not a header

Reply:
The icon isn't ready yet so I used `cancel` instead

Reply:
Please change it from a header to a button, and you can leave the cancel text in there